Modify the comments for pinned state.

alarshi opened the pull request:
This PR adds the correct values for grain growth test case using the pinned state flow.

Commit:
Modify the comments for pinned state.

jdannberg commented:
I am very confused by this. 

1. First, don't you need to update the test output as well? In the meeting today it sounded as if your PR included this as well, or did Ranpeng want to do that separately?
2. Second, I don't think the way you changed the units is correct; it needs to be m^(p-1) since A is multiplied with x (box extent in m) in the equation (see also: https://github.com/geodynamics/aspect/pull/5131). 
3. Third, can you explain how you got to the new value of A? We left the grain growth rate constant in the input file the same, so I though the only changes would be 

new_A = old_A * geometric_constant * phase_fraction_phase_one * phase_fraction_phase_two * sqrt(h_G)^(-grain_growth_exponent)

new_A = old_A * 3 * 0.4 * 0.6 * (2/pi)^(-10)
new_A = old_A * 65.85

What am I missing here?

Commit:
Address comments from the PR.

alarshi commented:
Thank you for the review! 
1) I should have added the current output of the test, just did it now in the latest commit.
2) Yes, I have now corrected the description based on your PR.
3) Oh my bad, the value was actually A*x, or old_A * 65.85 * 5000 (time of 5e4 years and velocity of 0.1), I have modified it in the latest commit. 

Thanks for catching these points!

jdannberg commented:
Okay, looks good to me! Thank you!
